packages/map-template/src/components/Search/Categories/Categories.jsx

```diff
@@ -151,19 +159,60 @@ function Categories({ onSetSize, getFilteredLocations, searchFieldRef, isOpen })
         setCategoriesWithoutChildKeys(topLevelCategories);
     }, [categories]);
 
+    /**
```

Comment:
```suggestion
    /*
```

Not a function definition, but a function call.